docs/import/manage_layers.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Manage Layers</title>
+  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): This is pinned to a particular version, but we should be targeting the live
+       version, whatever it is. -->
+  <script src="https://rawcdn.githack.com/google/earthengine-api/3bb86bfc4f3d9eed98220f3d225b414982915b86/javascript/build/ee_api_js_debug.js"></script>
+  <!-- TODO(janakr): download this script to lib. -->
+  <script src="https://apis.google.com/js/api.js"></script>
+
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-app.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-firestore.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.2.1/firebase-auth.js"></script>
+  <!-- Load jQuery, a 3rd-party library. -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/jquery-ui.js"></script>
+
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script type="module" src="manage_layers_startup.js"></script>
+
+  <!-- Load our styles. -->
+  <link rel="stylesheet" href="../style.css">
+  <link rel="stylesheet" href="manage_layers.css">
+</head>
+<body>
+
+<div id="navbar"></div>
+
+<br><br><br>  <!-- TODO(ramvellanki): switch to using a margin above the main content of this page -->
+
+<div id="disaster-picker">
+  <label for="disaster">Disaster: </label>
+  <select id="disaster" style="width: 160px"></select>
+  <select id="pending-disaster" style="width: 160px" hidden="true" disabled="true">
```

Comment:
This starts as hidden and I think is only ever .hide()-ed. Should this start as shown?

Reply:
Nice catch! Also there was some weirdness with the other select. I think all fixed.